Project.toml

```diff
@@ -43,6 +55,7 @@ NaNMath = "0.3, 1"
 Requires = "1.1"
 SnoopPrecompile = "1.0.3"
 SpecialFunctions = "1.6, 2"
+Tracker = "0.2"
```

Comment:
It seems that these 3 packages should be added to [extras] as well to make CI happy

Reply:
Yep, I added that.